config/initializers/config.rb

```diff
@@ -0,0 +1,36 @@
+Config.setup do |config|
+  # Name of the constant exposing loaded settings
+  config.const_name = 'Settings'
```

Comment:
also I don't think we need this anymore, since rails ships with a secrets store (the one I linked in the issue)